packages/discovery-provider/plugins/pedalboard/apps/solana-relay/src/routes/relay/relay.ts

```diff
@@ -219,8 +226,27 @@ export const relay = async (
       feePayer: feePayerKey.toBase58()
     })
 
+    const isGeoDataTransaction = shouldAttachGeoData(decompiled.instructions)
+    if (isGeoDataTransaction) {
+      const geo = await getRequestIpData(res.locals.logger, req)
+      if (geo) {
+        const memoInstruction = new TransactionInstruction({
+          keys: [{ pubkey: feePayerKey, isSigner: true, isWritable: true }],
+          data: Buffer.from(`geo:${JSON.stringify(geo)}`, 'utf-8'),
```

Comment:
nit: Wondering if this should be defined as a constant or explicitly named function somewhere to make it very obvious what the format is?

Reply:
done!